### PR TITLE
(chore): convert all underscores to hyphens in URLs

### LINKED ISSUE #34: Transition slugs to hyphenate (vs underscore)
For SEO optimization: 
- hyphens are best practices for slugs
- example: `https://quickstarts.postman.com/guide/postman_flows_tutorials/index.html` should be `https://quickstarts.postman.com/guide/postman-flows-tutorials/index.html`

- [ ] Migrate existing pages to hyphens
- [ ] Confirm new templates are built properly
- [ ] Update README and other documentation to require hyphens
- [ ] Update Vale guidelines (if applicable) to conform to new standard

### PR DESCRIPTION
**Pull request by Gbahdeyboh**
This PR fixes #34.

It modifies all the markdown files to use underscores and includes a script that dynamically does that incase it was skipped.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ export PATH=$PATH:$HOME/go/bin
 1. Terminate the running server with `ctrl C` and and navigate to the `pmguides` source directory `cd pmguides/src`
    - In this directory, you will see all existing guides and their markdown files.
 2. Generate a new guide from the guide template `npm run template <GUIDE_NAME>`
-   - Don't use spaces in the name of your guide, instead use underscores.
+   - Don't use spaces in the name of your guide, instead use **hyphens** to seperate words.
```
Comment (loopDelicious) [inline README.md:75]: @Gbahdeyboh typo, should be "separate"

**Review by loopDelicious — CHANGES_REQUESTED** [review]
@Gbahdeyboh can you also confirm the md filenames don't need to be updated?

**Comment by Gbahdeyboh** [comment]
Yes, they don't. Just their ID needs to be updated.

File: site/gulpfile.js [hunk 2]
```diff
@@ -638,7 +638,14 @@ const generateView = () => {
 
     // Get the list of codelabs and categories for this view
     const filtered = filterCodelabs(view, all.codelabs);
-    const codelabs = filtered.codelabs;
+    let codelabs = filtered.codelabs;
+    // Convert all codelab IDs from underscore(_) to dash(-)
+    codelabs = codelabs.map(codelab => {
+      codelab.id = codelab.id.split('_').join('-')
+      codelab.url = codelab.url.split('_').join('-')
```
Comment (Gbahdeyboh) [inline site/gulpfile.js:645]: @loopDelicious this line also ensures that the URL always uses hyphen even if the id uses an underscore.

File: README.md (hunk 1 above)
Comment (Gbahdeyboh) [inline README.md:75]:
```suggestion
   - Don't use spaces in the name of your guide, instead use **hyphens** to separate words.
```

Comment (Gbahdeyboh) [reply README.md]: Fixed!

**Pull request merged**